src/tests/class_tests/openms/source/SVMWrapper_test.cpp

```diff
@@ -1051,8 +1052,10 @@ START_SECTION((void getSVCProbabilities(struct svm_problem *problem, std::vector
 	TEST_EQUAL(predicted_labels.size() == probabilities.size(), true)
 	for (Size i = 0; i < predicted_labels.size(); ++i)
 	{
-		TEST_EQUAL((predicted_labels[i] < 0 && probabilities[i] < 0.5)
-							|| (predicted_labels[i] > 0 && probabilities[i] >= 0.5), true)
+		// At probability 0.5, LibSVM will assign the first encountered label in the training data
+		// in this case "-1"
+		TEST_EQUAL((predicted_labels[i] < 0 && probabilities[i] <= 0.5)
```

Comment:
Cool. Thanks for checking. I was always assuming this is related to a specific version!